linebot/models/responses.py

```diff
@@ -43,6 +43,28 @@ def __init__(self, display_name=None, user_id=None, picture_url=None,
         self.status_message = status_message
 
 
+class MemberIdsResponse(Base):
```

Comment:
Other classes in this module `responses` does not have such suffix `Response`. How about remove the suffix and keep `MemberIds` simply?

Reply:
Thanks for your feedback. I have fixed.